app/preprints/index/template.hbs

```diff
@@ -0,0 +1,160 @@
+{{page-title (t 'preprints.title')}}
+
+<div local-class='preprints-page-container' {{with-branding (get-model this.theme.provider.brand) }}>
+
+    {{!HEADER}}
+    <div local-class='preprint-header'>
+        <BrandedHeader
+            @translationParent='preprints'
+            @onSearch={{action this.onSearch}}
+            @showHelp=true
+            as |branded-header|
+        >
+            {{#branded-header.lead}}
+                <div local-class='description'>
+                    {{html-safe this.theme.provider.description}}
+                </div>
+                <div>
+                    <OsfLink
+                        data-test-poweredby-button
+                        data-analytics-name='Preprints powered-by button'
```

Comment:
If you have the `data-analytics-scope` above, you shouldn't need to add 'Preprints' to each of these `data-analytics-name`s. Unless something changes with the new analytics, of course.